scp.py

```diff
@@ -200,7 +200,8 @@ def get(self, remote_path, local_path='',
 
     def _open(self):
         """open a scp channel"""
-        if self.channel is None:
+        if self.channel is None or \
+                self.channel.closed is True:
```

Comment:
Looks cleaner. I had added the line break in the apprehension of PEP8 habit; that the line length may go over 80 chars but now when I checked, it hasn't. Thank you for the suggestion.